MSCL/source/mscl/MicroStrain/MIP/MipNodeFeatures.h

```diff
@@ -265,5 +265,19 @@ namespace mscl
         //Returns:
         //    A <GpioPinOptions> map of supported GPIO pin configurations
         const GpioPinOptions supportedGpioConfigurations() const;
+
+        //API Function: getEventActionInfo
+        //    Gets <EventSupportInfo> for event triggers and their types supported by the device.
+        //
+        //Returns:
+        //    <EventSupportInfo> of the event actions and their types
+        const EventSupportInfo getEventActionInfo() const;
```

Comment:
suggested rename: supportedEventActionInfo, supportedEventActionTypes?
apply same to getEventTriggerInfo

Reply:
This returns the types and info about them. I think keeping info in there makes the most sense. supportedEventActionTypeInfo could work, but that's a bit long